systems/plants/ContactForceTorqueSensor.m

```diff
@@ -123,7 +123,7 @@
         nD = length(tsmanip.LCP_cache.data.contact_data.d);
         nC_body = nA + nB;
         nC = size(tsmanip.LCP_cache.data.contact_data.normal,2);
-        nL = manip.getNumJointLimitConstraints;
+        nL = sum(tsmanip.LCP_cache.data.possible_limit_indices);
         nP = 2*manip.num_position_constraints;  % number of position constraints
```

Comment:
Got it, no worries -- sounds like this hasn't been properly tested in a
while. I can make the actual change whenever I'm next sitting down.
On Feb 4, 2015 7:00 PM, "Russ Tedrake" <notifications@github.com> wrote:

> In systems/plants/ContactForceTorqueSensor.m
> <https://github.com/RobotLocomotion/drake/pull/740#discussion_r24132282>:
>
> > @@ -123,7 +123,7 @@
> >          nD = length(tsmanip.LCP_cache.data.contact_data.d);
> >          nC_body = nA + nB;
> >          nC = size(tsmanip.LCP_cache.data.contact_data.normal,2);
> > -        nL = manip.getNumJointLimitConstraints;
> > +        nL = sum(tsmanip.LCP_cache.data.possible_limit_indices);
> >          nP = 2*manip.num_position_constraints;  % number of position constraints
>
> another good catch. i changed formulAtions a few months ago which halved
> the number of position constraints required. sorry for missing this one,
> too - Russ
> … <#14b5709911bfe921_>
>  On Feb 4, 2015, at 4:44 PM, Gregory Izatt <notifications@github.com>
> wrote: In systems/plants/ContactForceTorqueSensor.m: > @@ -123,7 +123,7 @@
> > nD = length(tsmanip.LCP_cache.data.contact_data.d); > nC_body = nA + nB;
> > nC = size(tsmanip.LCP_cache.data.contact_data.normal,2); > - nL =
> manip.getNumJointLimitConstraints; > + nL =
> sum(tsmanip.LCP_cache.data.possible_limit_indices); > nP =
> 2*manip.num_position_constraints; % number of position constraints Actually
> -- should the 2x multiplier still be here? It does not appear in the
> production of nP over in the solve LCP function in the TSRBM, and it's been
> causing trouble for me as I'm trying to move this code over to apply for
> the FT sensor on the arm. — Reply to this email directly or view it on
> GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/740/files#r24132282>.
>